README.md

```diff
@@ -5,11 +5,25 @@
  
  ## Introduction
 
-This is the Swiss implementation of the Electronic Health Certificates (EHN) Specification [[1](https://github.com/ehn-digital-green-development/hcert-spec)] used to verify the validity of Digital Covid Certificates. It is partly based on the reference implementation of EHN's `ValidationCore` [[2](https://github.com/ehn-digital-green-development/ValidationCore/tree/main/Sources/ValidationCore)]. 
- 
- ## Work in Progress
+This is the implementation of the [Electronic Health Certificates (EHN) specification](https://github.com/ehn-digital-green-development/hcert-spec)
+used to verify the validity of COVID Certificates [in Switzerland](https://github.com/admin-ch/CovidCertificate-App-Android).
+
+It is partly based on the reference implementation of EHN's `ValidationCore` [[2](https://github.com/ehn-digital-green-development/ValidationCore/tree/main/Sources/ValidationCore)]. 
+
+## Contribution Guide
+
+This project is truly open-source and we welcome any feedback on the code regarding both the implementation and security aspects.
+
+Bugs or potential problems should be reported using Github issues.
+We welcome all pull requests that improve the quality of the source code.
 
-The CovidCertificateSDK for iOS is work in progress and is not yet complete. We are continuing the development of this library, and the API is likely to change. Please contact the BIT before using / integrating this SDK in your app.
+## Repositories
+
+* iOS App: [CovidCertificate-App-iOS](https://github.com/admin-ch/CovidCertificate-App-iOS)
+* Android App: [CovidCertificate-App-Android](https://github.com/admin-ch/CovidCertificate-App-Android)
+* Androiid SDK: [CovidCertificate-SDK-Android](https://github.com/admin-ch/CovidCertificate-SDK-Android)
+* Verifier Service: [CovidCertificate-App-Verifier-Service](https://github.com/admin-ch/CovidCertificate-App-Verifier-Service)
```

Comment:
Just delete the verifier service line, the general note below is enough I think. Let's only explicitly link the app-facing repos.